scripts/cookbook_parity_check.py

```
@@ -63,7 +63,14 @@ def rule_reset1(df):
 
 
 def rule_sched1(df):
-    return df["occ_mode"].eq("unoccupied") & df["fan_status"].astype(bool)
+    occ = df["occ_mode"]
+    occ_str = occ.astype(str).str.strip().str.lower()
+    label = occ_str.isin(
+        {"unoccupied", "unocc", "off", "false", "night", "standby", "setback", "no"}
+    )
+    num = pd.to_numeric(occ, errors="coerce")
```

Ruff (0.16.1)

[error] Undefined name `pd`

(F821)